spec/models/mention_spec.rb

```diff
@@ -25,6 +24,25 @@
     end
   end
 
+  describe "#mention_syntax" do
+    before do
+      @user = alice
+      @aspect1 = @user.aspects.create(:name => 'second_aspect')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.